Assets/Source/Entities/Behaviors/Commander.cs

```diff
@@ -7,9 +7,11 @@ public class Commander : MonoBehaviour
 {
 	// Get rid of these after testing movement
 	private GameObject tempCircle;
+	private Player player;
 
 	void Start()
 	{
+		player = transform.root.GetComponent<Player>();
```

Comment:
One reason we might want to separate them out in some cases is because, in Unity, transform due to animation can most easily be overridden/controlled by childing the animated mesh to an empty which represents the position of the player. This is only a single reason though. It's quite possible we'll need this type of flexibility for other reasons. Ultimately, we just have to decide how we're going to arrange things in the scene, and this will drive a lot of our inter-component communication.